LibraryApp/src/redux/auth/actions.js

```diff
@@ -24,14 +25,32 @@ const privateActionCreators = {
       type: actions.LOGIN_FAILURE,
       payload: problem
     });
+  },
+  initWithStoredUser: () => dispatch => {
+    dispatch({
+      type: actions.INIT_STORED_USER
+    });
+    dispatch(
+      NavigationActions.navigate({
+        routeName: Routes.Library
+      })
+    );
   }
 };
 
 const actionCreators = {
+  initRemembered: () => dispatch => {
+    dispatch(privateActionCreators.initWithStoredUser());
+  },
   login: (email, password) => async dispatch => {
     dispatch({ type: actions.LOGIN });
     const response = await loginService(email, password);
-    if (response?.data?.data) {
+    if (response.ok) {
+      const { headers } = response;
+      const token = headers['access-token'];
+      const { client, uid } = headers;
+      // debugger;
+      await setCurrentUserStorage(token, client, uid);
```

Comment:
this function should receive only `response.headers` and transform the object inside